From db93f78c3e59cdd78713f93eb14fc7b965fefe19 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 6 Mar 2021 21:51:32 +0000
Subject: [PATCH] gnu: Add cl-mount-info

* gnu/packages/lisp-xyz.scm (sbcl-cl-mount-info, cl-mount-info,
  ecl-cl-mount-info): New variables.
---
 gnu/packages/lisp-xyz.scm | 32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 161a15c44f..48af024b17 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -15062,3 +15062,35 @@ Lisp.")
 
 (define-public cl-percent-encoding
   (sbcl-package->cl-source-package sbcl-percent-encoding))
+
+(define-public sbcl-cl-mount-info
+  (let ((commit "2024f5037a7f63db3e3587dc9972cd7b9318f06b")
+        (revision "1"))
+    (package
+     (name "sbcl-cl-mount-info")
+     (version (git-version "0.0.1" revision commit))
+     (source
+      (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://notabug.org/cage/cl-mount-info.git")
+             (commit commit)))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0vza9gj9q42nzb5v8aj22lmn4aqx9vrddsb5a343nbwfz89hbh9x"))))
+     (build-system asdf-build-system/sbcl)
+     (inputs
+      `(("alexandria" ,sbcl-alexandria)
+        ("cffi" ,sbcl-cffi)
+        ("cl-ppcre" ,sbcl-cl-ppcre)))
+     (home-page "https://notabug.org/cage/cl-mount-info.git")
+     (synopsis "Common Lisp wrapper around getmntent(3) and related C functions")
+     (description
+      "CL-MOUNT-INFO allows to get information about the mounted file system.")
+     (license license:gpl3))))
+
+(define-public ecl-cl-mount-info
+  (sbcl-package->ecl-package sbcl-cl-mount-info))
+
+(define-public cl-mount-info
+  (sbcl-package->cl-source-package sbcl-cl-mount-info))
-- 
2.29.2

